addons/default/visiosoft/advs-module/resources/js/new-create.js

```diff
@@ -210,11 +210,15 @@ $(document).ready(function () {
 
     });
 
-    $(".priceField, .priceDecimalField").on('change', function () {
+    $(".priceField, .priceDecimalField, .standard-price-field, .standard-price-decimal-field").on('change', function () {
         let price = $(".priceField").val() === "" ? '0' : $(".priceField").val();
```

Comment:
this way far from the merge request title I guess. But !

Reply:
This merge request consist of two commits, check the commit for more precise title ;)